src/components/structures/TimelinePanel.js

```diff
@@ -108,7 +108,7 @@ var TimelinePanel = React.createClass({
 
     getDefaultProps: function() {
         return {
-            timelineCap: 250,
+            timelineCap: 1000,
```

Comment:
hum; doesn't this just mean we'll hit the same problem when we get to 1000 invisible events? I'd advise finding a way to disable the cap altogether. (Possibly just by setting it to Number.MAX_VALUE)